src/TestHelper/dir.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I agree with @stephentoub as well.  CoreFx.TestUtilties is fine by me.  As we had discussed before you could consider using Private or Internal in the name if you wanted to discourage its use by product assemblies.